src/outagesim/backup_reliability.jl

```diff
@@ -230,91 +462,99 @@ function shift_gen_battery_prob_matrix!(gen_battery_prob_matrix::Matrix, shift_v
 end
 
 """
-    survival_over_time_gen_only(critical_load::Vector, OA::Real, failure_to_start::Real, failure_to_run::Real, num_generators::Int,
-                                gen_capacity::Real, max_duration::Int; marginal_survival = true)::Matrix{Float64}
+    survival_gen_only(;critical_load::Vector, gen_operational_availability::Real, failure_to_start::Real, failure_to_run::Real, num_generators::Int,
+                                gen_capacity_kw::Real, max_duration::Int, marginal_survival = true)::Matrix{Float64}
 
-Return a matrix of probability of survival with rows denoting outage start and columns denoting outage duration
+Return a matrix of probability of survival with rows denoting outage start timestep and columns denoting outage duration
 
 Solves for probability of survival given only backup generators (no battery backup). 
-If ``marginal_survival`` = true then result is chance of surviving in given outage hour, 
-if ``marginal_survival`` = false then result is chance of surviving up to and including given outage hour.
+If ``marginal_survival`` = true then result is chance of surviving in given outage timestep, 
+if ``marginal_survival`` = false then result is chance of surviving up to and including given outage timestep.
 
 # Arguments
-- `critical_load::Vector`: 8760 vector of system critical loads. 
-- `OA::Real`: Operational Availability of backup generators.
-- `failure_to_start::Real`: probability of generator Failure to Start and support load. 
-- `failure_to_run::Real`: hourly Failure to Run probability. failure_to_run is 1/MTTF (mean time to failure). 
-- `num_generators::Int`: number of generators in microgrid.
-- `gen_capacity::Real`: size of generator.
+- `critical_load_kw::Vector`: 8760 vector of system critical loads. 
+- `gen_operational_availability::Union{Real, Vector{<:Real}}`: Operational Availability of backup generators.
+- `failure_to_start::Union{Real, Vector{<:Real}}`: probability of generator Failure to Start and support load. 
+- `failure_to_run::Union{Real, Vector{<:Real}}`: hourly Failure to Run probability. failure_to_run is 1/MTTF (mean time to failure). 
+- `num_generators::Union{Int, Vector{Int}}`: number of generators in microgrid.
+- `gen_capacity_kw::Union{Real, Vector{<:Real}}`: size of generator.
 - `max_duration::Int`: maximum outage duration in hours.
```

Comment:
Is only 8760 supported so timesteps=hours? If so, that should be documented more clearly.

Reply:
@sericson0 can you address this? You replaced most "hours" with "time_steps", but I still see one, as well as documentation that says certain vectors must be 8760s.

Reply:
Pushed an update to make replacements

Reply:
I'm still seeing references to 8760